src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -1008,6 +1008,13 @@ To use a custom JDK path for a command line build, set the 'JavaSdkDirectory' MS
     <comment>The following are literal names and should not be translated: MSBuild, .NET.
 {0} - The deprecated MSBuild property name
 {1} - The numeric version of .NET</comment>
+  </data>
+  <data name="XA1038" xml:space="preserve">
+    <value>The '{0}' MSBuild property has an invalid value of '{1}''. Value values are '{2}'.</value>
+    <comment>The following are literal names and should not be translated: MSBuild.
+{0} - The MSBuild property name which has invalid values.
+{1} - The current value of the property
+{2} - The list of valid values.</comment>
```

Comment:
What *kind* of list?  Is a comma-separated list even valid across most languages?

Would be useful to look for some localization guidance for this type of pattern.